runtime/mux.go

```diff
@@ -185,7 +185,7 @@ func (s *ServeMux) HandlePath(meth string, pathPattern string, h HandlerFunc) er
 		return fmt.Errorf("parsing path pattern: %w", err)
 	}
 	tp := compiler.Compile()
-	pattern, err := NewPattern(tp.Version, tp.OpCodes, tp.Pool, tp.Verb)
+	pattern, err := NewPattern(tp.Version, tp.OpCodes, tp.Pool, tp.Verb, false)
```

Comment:
default to false in the HandlePath implementation for now. deferring to later commits or PR for this